get.sh

```diff
@@ -95,7 +95,7 @@ getBinaryOpenjdk()
 getTestKitGen()
 {
 	cd $TESTDIR
-	git clone https://github.com/eclipse/openj9.git
+	git clone https://github.com/Mesbah-Alam/openj9.git -b addExportForJvmVersion
```

Comment:
This should be updated.

Reply:
Yep, done. I had only added it to test it off of my branch in a personal build.